app/controllers/api/v4/converts_controller.rb

```diff
@@ -7,6 +7,13 @@ def create
 
     @run = Run.new(run_file: run_file, user: nil)
     @run.parse(fast: ["on", "1"].include?(params[:historic]) ? false : true, convert: true)
+    unless @run.splits.present?
```

Comment:
`[].present?` returns `false`, but `[]` is a valid outcome for `@run.splits` after parsing.